pubspec.yaml

```diff
@@ -8,8 +8,8 @@ author: Dart Team <misc@dartlang.org>
 environment:
```

Comment:
From another thread, @nshahan wrote:

> Homepage -> https://webdev.dartlang.org/components
> Docs -> https://webdev.dartlang.org/api?package=angular_components

Will you be adding the `documentation` entry in this release?